components/carousel/module/CarouselAgnostic.vue

```diff
@@ -95,6 +122,32 @@ const totalDots = computed(() => {
 const dotHelper = computed(() =>
   slider.value && totalDots.value > 0 ? [...Array(totalDots.value).keys()] : []
 )
+
+const slideIndex = ref(3)
+
+const sliderSettings = computed(() => {
+  if (slider.value) {
+    const { track, options, slides } = slider.value
+    const abs = track.details.abs
```

Comment:
is this a number?

Reply:
not sure what are you hinting at here, are there some types missing?

Reply:
`track.details.abs` has any type, afterward you use it in a `Math.min` & `Math.max` do you know if there could be side effects?

Reply:
you're right, sorry, added Number constructor to abs and perView to hopefully avoid some of the unintended sideffects